src/libraries/Pool.sol

```diff
@@ -325,7 +325,7 @@ library Pool {
             if (params.sqrtPriceLimitX96 >= slot0Start.sqrtPriceX96()) {
                 PriceLimitAlreadyExceeded.selector.revertWith(slot0Start.sqrtPriceX96(), params.sqrtPriceLimitX96);
             }
-            if (params.sqrtPriceLimitX96 < TickMath.MIN_SQRT_PRICE) {
+            if (params.sqrtPriceLimitX96 <= TickMath.MIN_SQRT_PRICE) {
```

Comment:
Note: This is an unrelated change to sb 43 and 44. The relevant changes are in TickMath.sol